eureka/S5_lightcurve_fitting/lightcurve.py

```diff
@@ -109,31 +109,41 @@ def fit(self, model, fitter='lsq', **kwargs):
         # else:
         #     for n in range(len(model.components)):
         #         model.components[n].time = self.time
-
+        
         if fitter == 'lmfit':
 
             # Run the fit
-            fit_model = f.lmfitter(self.time, self.flux, model, self.unc, **kwargs)
+            fit_model = f.lmfitter(self.time, self.flux, model, meta, self.unc, **kwargs)
 
         elif fitter == 'demc':
 
             # Run the fit
-            fit_model = f.demcfitter(self.time, self.flux, model, self.unc, **kwargs)
+            fit_model = f.demcfitter(self.time, self.flux, model, meta, self.unc, **kwargs)
 
         elif fitter == 'lsq':
 
             # Run the fit
-            fit_model = f.lsqfitter(self, model, **kwargs)
+            fit_model = f.lsqfitter(self, model, meta, **kwargs)
             #fit_model = f.lsqfitter(self.time, self.flux, model, self.unc, **kwargs)
 
+        elif fitter == 'emcee':
+
+            # Run the fit
+            fit_model = f.emceefitter(self, model, meta, **kwargs)
+
+        elif fitter == 'dynesty':
+
+            # Run the fit
+            fit_model = f.dynestyfitter(self, model, meta, **kwargs)
+
         else:
             raise ValueError("{} is not a valid fitter.".format(fitter))
 
         # Store it
         if fit_model is not None:
             self.results.append(fit_model)
 
-    def plot(self, fits=True, draw=True):
+    def plot(self, meta, fits=True, draw=True):
```

Comment:
Might be time to update the name of this function from `plot` to `plotLC`, or something similarly descriptive.

Reply:
Well the function is already a class method, so it gets used like `lc = LightCurve(...); lc.plot(...)` which seems clear enough to me. We can still rename it if you'd like though

Reply:
Good point!